lib/genome/online_updaters/docm/api_client.rb

```diff
@@ -0,0 +1,29 @@
+module Genome; module OnlineUpdaters; module Docm
+  class ApiClient
+    def variants
+      @variants = JSON.parse(get_page(variant_url))
+    end
+
+    def variant_url
+      'http://docm.genome.wustl.edu/api/v1/variants.json'
+    end
+
+    def get_page(url)
+      uri = URI(url)
+
+      uri.query = URI.encode_www_form(docm_params)
+
+      req = Net::HTTP::Get.new(uri)
+      resp = Net::HTTP.start(uri.host, uri.port) { |http| http.read_timeout = 1000; http.request(req)}
```

Comment:
I'm guessing you're using this form rather than `res = Net::HTTP.get_response(uri)` because the endpoint is slow and you need to specify the timeout manually?